Source/WebInspectorUI/UserInterface/Models/Font.js

```diff
@@ -25,24 +25,33 @@
 
 WI.Font = class Font
 {
-    constructor(name, variationAxes)
+    constructor(name, variationAxes, {syntheticBold, syntheticOblique} = {})
     {
         this._name = name;
         this._variationAxes = variationAxes;
+        this._syntheticBold = syntheticBold;
+        this._syntheticOblique = syntheticOblique;
```

Comment:
Since these are optional (since they're inside a `{...} = {}`) we should have the fallback behavior be here instead of inside `fromPayload`.
```
this._syntheticBold = !!syntheticBold;
this._syntheticOblique = !!syntheticOblique;
```

also ditto ([Source/JavaScriptCore/inspector/protocol/CSS.json](#diff-e4ce9b37fd298901a223600e617fea37991c6ceaa01e238a5cdb9dd67d7190f8):257)